src/core/schema.js

```diff
@@ -138,6 +138,7 @@ function parseProperty (value, propDefinition) {
   // Use default value if value is falsy.
   if (value === undefined || value === null || value === '') {
     value = propDefinition.default;
+    if (Array.isArray(value)) { value = value.slice(); }
```

Comment:
That's more of a weird workaround. It's not really parsing, I'd have to run a check if it's exactly the default object to figure to copy it, and I only need it to be done once if it's using the default value and not on every parse. I propose to handle it because the case is unexpected.

The object copy won't be hit that often if we handle the logic here. Only once per property initialization, and only if it's usually the default value. Plus, it's usually an empty object.